server.js

```diff
@@ -9,29 +9,48 @@ var bot = controller.spawn({
   retry: 20
 }).startRTM()
 
-var ls_greet = "Welcome to LocalSupport! :slightly_smiling_face: To install the codebase see: https://github.com/AgileVentures/LocalSupport/blob/develop/docs/installation.md#installation and our #localsupport-install channel if you need help"
+var ls_greet = "Welcome to LocalSupport <@USER_NAME> :slightly_smiling_face: To install the codebase see: https://github.com/AgileVentures/LocalSupport/blob/develop/docs/installation.md#installation and our #localsupport-install channel if you need help"
 
 ls_greet += "\n\n\n see https://github.com/AgileVentures/LocalSupport/blob/develop/CONTRIBUTING.md for how to contribute to the project, but feel free to ask questions here too :-)"
 
-var wso_greet = "Welcome to WebSiteOne! :slightly_smiling_face: To install the code see: https://github.com/AgileVentures/WebsiteOne/blob/develop/docs/project_setup.md and our #websiteone-install channel if you need help"
+var wso_greet = "Welcome to WebSiteOne <@USER_NAME> :slightly_smiling_face: To install the code see: https://github.com/AgileVentures/WebsiteOne/blob/develop/docs/project_setup.md and our #websiteone-install channel if you need help"
 
 wso_greet += "\n\n\n feel free to ask questions here, but do also see https://github.com/AgileVentures/WebsiteOne/blob/develop/CONTRIBUTING.md for contribution guidelines :-)"
-// for testing on staging slack
+
+var rfm_greet = "Welcome <@USER_NAME> to #rundfunk-mitbestimmen!"
+
+rfm_greet += "\n\nLearn about the goal of this project by visiting our website"
+rfm_greet += "\nhttps://rundfunk-mitbestimmen.de/"
+rfm_greet += "\nand our Github page"
```

Comment:
the formatting for links in slack is supposed to be `So does this one: <http://www.foo.com|www.foo.com>` From https://api.slack.com/docs/message-formatting#linking_to_urls